### ISSUE TITLE
Missing Buttons in GUI

### ISSUE THREAD

**Issue by jmtw000** [AppleCommander/AppleCommander#31]
On Windows 10, when I launch the GUI the buttons are missing. I have uninstalled and reinstalled the latest version of Java but the problem persists. Running from the command line there are no errors reported. Any idea what would cause this?
![1](https://user-images.githubusercontent.com/46089724/50385756-a4c29a00-06a8-11e9-91ed-cd0841d50bf2.jpg)

**Comment by a2geek** [comment]
(Sorry, I tend to go in fits and starts.)

Not particularly certain on this. In case it matters, I usually develop on my Linux desktop. But I do have access to the other 2 platforms. I'll see what I can identify. Beyond me building without the physical images in the JAR file, my guess would be in different version of the SWT windowing library.

**Comment by a2geek** [comment]
I just checked on my Mac and Windows 10 devices and I everything looks fine. Is this still an issue? Making a stab at the dark ... what version of Java?  And I assume your Windows 10 is up-to-date? The JAR file does appear to have the images in it as well as the SWT DLL files, so I think it's fine from that aspect...

**Comment by jmtw000** [comment]
I was having the issue of the missing buttons on one Windows 10 machine and not on another. Today, I started comparing java versions, settings, etc. Everything was identical. Then I remembered the machine with the issue had the display scaling set to 300% because it's connected to a large display that's difficult to read at 100%. Once I removed the scaling I could see the buttons again! Thanks for looking into this.
